pretext/Classes/Tamagotchi.ptx

```diff
@@ -134,6 +182,54 @@
             import sys
             sys.setExecutionLimit(60000)
 
+            from random import randrange
```

Comment:
The interface here doesn't work very well on my browser, I can't see the output and the input prompt together.